spec/features/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require './spec/web_helpers'
+
```

Comment:
Naming of feature tests should be more like 'doing something' e.g. adding_peep_spec, viewing_peeps_spec